src/app/features/characters/components/character-info/character-info.component.scss

```diff
@@ -0,0 +1,59 @@
+@import '../../../../styles/button.scss';
```

Comment:
try to implement stylePreprocessorOptions in angular.json, it will allow you to import styles without ../../../

https://angular.io/guide/workspace-config#style-preprocessor-options

Reply:
I will change it in the next PR